integrations/selinux/teleport.fc

```diff
@@ -0,0 +1,4 @@
+/usr/local/bin/teleport  -- gen_context(system_u:object_r:teleport_exec_t,s0)
+/run/.teleport.pid(\d+)  -- gen_context(system_u:object_r:teleport_pid_t,s0)
+/etc/teleport.yaml       -- gen_context(system_u:object_r:teleport_conf_t,s0)
+/var/lib/teleport(/.*)?     gen_context(system_u:object_r:teleport_data_t,s0)
```

Comment:
This list is missing the managed updates v1 updater's config directory, else this will break the old AU system: `/etc/teleport-upgrade.d(/.*)`

Reply:
This list is also missing the ability to read the updater status file: `/opt/teleport/<installation-namespace>/update.yaml` which is used to detect and report the updater status.

This feature is not in v17 yet but will be in a few weeks.

Reply:
Good catch, I'll add the v1 updater config dir. The updater is what will be managing the policy and so the policy will not apply to it, so that's not needed. Writing an SELinux policy for a program and allowing it to change SELinux policies doesn't really do much, or make much sense.

Reply:
If I understand correctly, the policy applies to the Teleport agent. The Teleport agent needs to read the updater status file to report the updater status via its inventory stream.